fastlane_core/spec/core_ext/shellwords_ext_spec.rb

```diff
@@ -385,7 +385,7 @@ def expect_correct_implementation_to_be_called(obj, method, os)
       expect(WindowsShellwords).to receive(:shellescape).with(obj).and_call_original
       expect(Shellwords).not_to(receive(:escape))
     else
-      # Shellswords.escape
+      # Shellwords.escape
       expect(Shellwords).to receive(:escape).with(obj).and_call_original
```

Comment:
I suspect I couldn't figure out the intent when I looked the last time. I'm a bit more certain this time around.

Reply:
🐚 ⚔️😂